pkg/agent/route/route_linux.go

```diff
@@ -62,6 +63,8 @@ var (
 	// globalVMAC is used in the IPv6 neighbor configuration to advertise ND solicitation for the IPv6 address of the
 	// host gateway interface on other Nodes.
 	globalVMAC, _ = net.ParseMAC("aa:bb:cc:dd:ee:ff")
+	// iptablesInited is used to determine when iptables initialisation is done.
```

Comment:
s/initialisation/initialization

I'm not sure "inited" is an actual word, can we rename the variable `iptablesInitialized`?

Reply:
definitely, I was just wary that it might be too long.
Updated